### PR TITLE
Fix the PKG permission issue and change the install location

### LINKED ISSUE #492: Permissions issue in PKG installer
We've had a report that permissions were incorrectly set by PKG installer. Folks could not use the product without running `sudo chmod -R 555 /usr/local/share/dotnet`. Can you take a quick look?

### PR DESCRIPTION
**Pull request by blackdwarf**
A temporary fix for the PKG permission issue by using chmod -R 755 in the postinstall
script. Change the install location to remove the "cli" part of the path. Also
change the corehost to reflect this change.

Fixes #492 and #390 

/cc @piotrpMSFT @anurse @brthor

### CODE REVIEW

**Comment by Sridhar-MS** [comment]
:shipit:

**Comment by blackdwarf** [comment]
I think teh Windows build is failing with the AV again.

File: packaging/osx/scripts/postinstall [hunk 1]
```diff
@@ -4,6 +4,9 @@
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
 
+# A temporary fix for the permissions issue(s)
+chmod -R 755 $2
```
Comment (brthor) [inline packaging/osx/scripts/postinstall]: As discussed please make a comment here on what values $2 (and potentially $1) represent. May save some headache later :smile:

**Comment by brthor** [comment]
Small comment but otherwise :shipit:

File: packaging/osx/scripts/postinstall (hunk 1 above)
Comment (anurse) [inline packaging/osx/scripts/postinstall]: Better yet, start the script off by assigning `$1` and `$2` to named variables.

Comment (blackdwarf) [inline packaging/osx/scripts/postinstall]: @anurse that is a pretty good idea, actually. Though, for reference, the $1 to $n are pretty well defined for pkg creation on OS X.

File: packaging/osx/scripts/postinstall [hunk 2]
```diff
@@ -4,20 +4,27 @@
...
-ln -s $2/bin/dotnet-new /usr/local/bin/
-ln -s $2/bin/dotnet-pack /usr/local/bin/
-ln -s $2/bin/dotnet-publish /usr/local/bin/
-ln -s $2/bin/dotnet-repl /usr/local/bin/
-ln -s $2/bin/dotnet-restore /usr/local/bin/
-ln -s $2/bin/dotnet-resgen /usr/local/bin/
-ln -s $2/bin/dotnet-run /usr/local/bin/
-ln -s $2/bin/dotnet-test /usr/local/bin/
+PACKAGE=$1
+INSTALL_DESTINATION=$2
```
Comment (brthor) [inline packaging/osx/scripts/postinstall]: :+1:

**Comment by blackdwarf** [comment]
@anurse looks ok?

**Comment by anurse** [comment]
:shipit: as far as the corehost changes go

**Comment by blackdwarf** [comment]
Merging then.

**Comment by piotrpMSFT** [comment]
What's the long term fix?

**Comment by blackdwarf** [comment]
Do it the way Apple prescribes, if there is any. If not, then I would suggest this becomes the solution since it will work (just isn't super elegant). I'm still investigating what is the prescribed way.

**Pull request merged**